### PR TITLE
Add link to Katacoda, fix incomplete sentence

### PR DESCRIPTION
**Pull request by awang**
This change is [Reviewable](https://reviewable.io/reviews/materializeinc/materialize/3537)

### CODE REVIEW

**Comment by awang** [comment]
@sploiselle what do you think?  (Deploy preview: https://deploy-preview-3537--materializeinc.netlify.app/get-started/)

I much prefer embedding it in the web (https://deploy-preview-3538--materializeinc.netlify.app/get-started/), but we don't have enough width with the current layout

cc @rjnn

**Comment by benesch** [comment]
> I much prefer embedding directly instead of linking (https://deploy-preview-3538--materializeinc.netlify.app/get-started/), but we don't have enough width with the current layout

To be honest I think embedding it is always going to be pretty hard to use. Even if you put it on a page without a right sidebar (see https://deploy-preview-3538--materializeinc.netlify.app for example), the colors are still going to clash badly, and the scrolling and tabbing are pretty awkward.

Are you just worried about driving people away from materialize.io? If so we could look into getting it onto a page with the materialize.io logo + header but a full width (no docs sidebar) embed.

**Comment by quodlibetor** [comment]
If we could get it to start instantaneously with decent color scheme I think it might be pretty cool to just have it on the root materialize.io page, starting with an install/download though is rough.

**Comment by benesch** [comment]
We just need to throw some money Katacoda's way to make that happen: https://www.katacoda.community/custom-environment.html#multi-node-environments

**Comment by quodlibetor** [comment]
neat, yeah. I support us doing that long term, fwiw, but also definitely think that getting this into our docs soon is valuable, it's a really cool way to interact with materialize.

File: doc/user/content/get-started.md [hunk 1]
```diff
@@ -217,3 +219,7 @@ Once you're done, don't forget to stop `curl` and `rm wikirecent`.
 [Next, see how Materialize can work as an entire microservices →](/docs/demos/microservice)
 
 {{</ cta >}}
+
+## <a name="webdemo"></a> Web Demo
```
Comment (sploiselle) [inline doc/user/content/get-started.md:223]: I'd rather make this a CTA like the link right above it and replace the existing link to this section with it.

Also, just FYI that all of the headers automatically generate an anchor tag, so you don't need to create your own. If you preview the page and click the header, it'll give you the appropriate hash.

**Comment by sploiselle** [comment]
Just to second Nikhil's comment, embedding this on the page is pretty wonky irrespective of the rhs menu. The scrolling and tabbing strikes me as pretty heavy weight for an in-browser window, and the fact that it "autoplays" for everyone viewing the page feels a little too pushy. I think the current implementation is solid, though, by just linking to Katacoda directly.

I'd be on board with embedding it on the page if we can:
- Control its appearance
- Greatly reduce the time-to-interact
- Prevent auto-play

**Comment by awang** [comment]
Thanks everyone!  I'm going to try out this version (with a stronger CTA link) unless there are strong objections.

I agree with everything said!  I don't feel as strongly about disabling auto-play part, but we can discuss.  It looks like Mongo's web shell requires a click to enable, but Katacoda doesn't support it:  https://docs.mongodb.com/manual/tutorial/getting-started/

**Pull request merged**